src/clients/mocks/MockEvents.ts

```diff
@@ -34,8 +37,19 @@ const removeListener = (): void => {
 
 export class EventManager {
   private logIndexes: Record<string, number> = {};
+  public readonly minBlockRange = 10;
+  public readonly eventSignatures: Record<string, string> = {};
+
+  constructor(public blockNumber = 0) {}
 
-  constructor(public readonly eventSignatures: Record<string, string>) {}
+  addEventSignatures(eventSignatures: Record<string, string>): void {
```

Comment:
Should we apply an explicit visibility operator here? `public` `protected` `private` etc

Reply:
`Public` is the default so there's no change in behaviour as a result...is there any advantage to explicitly specifying it?